stdlib/2and3/xml/etree/ElementTree.pyi

```diff
@@ -53,8 +53,14 @@ class Element(MutableSequence[Element]):
     def extend(self, elements: Iterable[Element]) -> None: ...
     def find(self, path: _str_argument_type, namespaces: Optional[Dict[_str_argument_type, _str_argument_type]] = ...) -> Optional[Element]: ...
     def findall(self, path: _str_argument_type, namespaces: Optional[Dict[_str_argument_type, _str_argument_type]] = ...) -> List[Element]: ...
-    def findtext(self, path: _str_argument_type, default: Optional[_T] = ..., namespaces: Optional[Dict[_str_argument_type, _str_argument_type]] = ...) -> Union[_T, _str_result_type]: ...
-    def get(self, key: _str_argument_type, default: Optional[_T] = ...) -> Union[_str_result_type, _T]: ...
+    @overload
+    def findtext(self, path: _str_argument_type, namespaces: Optional[Dict[_str_argument_type, _str_argument_type]] = ...) -> Optional[_str_result_type]: ...
```

Comment:
Shouldn't namespaces be keyword-only in this overload?

Reply:
Yes, that's a good idea. Changed.